app/styles/components/scheduler.scss

```diff
@@ -14,17 +14,17 @@
   display: none;
 }
 
-.speaker {
-  &::after {
-    content: ', ';
-  }
+// .speaker {
```

Comment:
Where else was it being used? Ensure no other UI is impacted on deleting this

Reply:
figuring out that only